src/opm/parser/eclipse/EclipseState/IOConfig/IOConfig.cpp

```diff
@@ -193,6 +193,11 @@ namespace Opm {
         return m_UNIFOUT;
     }
 
+    void IOConfig::consistentFileFlags(){
```

Comment:
I was about to push the green button on this - but I don't understand the `consistentFileFlags()` function; seems unnecessary to me?

Reply:
It is need to set that the format of input and output is the same. Most runs to not set the inputformat since this does not matter since they are not intended to be restarted (the format is set in the restart deck.) With this change I removed one other change one other change in opm common. In flow I then call consistentFileFlags() to set the input file format correct when I restart from a file used for running the case.

Reply:
I understand; but I would like to not include this change in the final merge. If the deck is missing the required `UNIFIN` or `MULTIN` (I think it is ...) - that is a one shot operation to fix, and in my opinion better than to enforce something which might end up being in disagreement with what is in the deck.